tabbycat/participants/views.py

```diff
@@ -69,6 +70,7 @@ def get_context_data(self, **kwargs):
 
 
 class AdminParticipantsListView(AdministratorMixin, BaseParticipantsListView):
+    view_permission = Permission.VIEW_TEAMS and Permission.VIEW_ADJUDICATORS
```

Comment:
This won't work either. To annotate my earlier comment:
> I don't remember supporting a list of permissions, but it could be done easily. I'm assuming this is an `and`.

Meant that you can't have a view be dependent on multiple permissions, but giving a list of permissions could be supported by editing the mixin. You'd also need to specify that _all_ (and) of the listed permissions must be present, rather than merely _any_.

I'd rather you create a new permission to access this page, rather than try to do permission "logic".